tfbpmodeling/lasso_modeling.py

```diff
@@ -431,6 +437,16 @@ def get_modeling_data(
             )
             raise
 
+        if center_scale:
+            logger.info(f"Standardizing predictors (drop_intercept={drop_intercept})")
```

Comment:
```suggestion
            logger.info(f"Center model matrix = `{drop_intercept}`). Scale model matrix = `True`")
```

Reply:
note that we could pass in an option to set `with_std`. No need for that yet, hence setting `Scale model matrix = True` in the log for now if center_scale is True